src/helpers/db/fetch-counters.js

```diff
@@ -0,0 +1,10 @@
+async function fetchCounters() {
+  return Promise.resolve([
+    {
+      name: 'applicationId',
+      counter: 1
```

Comment:
If we're returning the new document in `generateApplicationId `, do we need to initialise it to be '0'?